css/navbar.css

```diff
@@ -51,6 +51,41 @@ header {
     z-index: var(--z-1);
 }
 
+/* Account Options */
+
+.dropbtn {
+    border: none;
+    cursor: pointer;
+}
+
+.dropdown {
+    position: relative;
+    display: inline-block;
+}
+
+.dropdown-content {
+    display: none;
+    position: absolute;
+    top: 3rem;
+    background-color: var(--light-grey);
+    min-width: 160px;
```

Comment:
can use rem here

Reply:
Yes, true. But here I didn't wanted it to be responsive hence used px instead of rem.